Plots/Polygons/NCL_polyg_18.py

```diff
@@ -0,0 +1,132 @@
+"""
+NCL_polyg_18.py
+==============
+This script illustrates the following concepts:
+   - Adding lines, markers, and polygons to a map
+   - Drawing lines, markers, polygons, and text in inset axes
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polyg_18.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polyg_18_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+# ----------------
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy
+from geocat.viz import util as gvutil
+import matplotlib.patches as mpatches
+from cartopy.mpl.ticker import LatitudeFormatter, LongitudeFormatter
+###############################################################################
+# Define helper function to remove ticks/frames from axes
+
+
+def removeTicks(axis):
+    axis.get_xaxis().set_visible(False)
+    axis.get_yaxis().set_visible(False)
+
+
+###############################################################################
+# Plot map, markers, and polygons
+
+# Set size of figure
+fig = plt.figure(figsize=(10, 10))
+
+# Make grid on figure with 2 rows, 1 column
+grid = plt.GridSpec(2, 20, hspace=0.2, wspace=0.2, figure=fig)
+
+# Make subplot for map
+ax = plt.subplot(grid[:-1, 1:], projection=ccrs.PlateCarree())
+
+# Add continents
+continents = cartopy.feature.NaturalEarthFeature(
+        name="coastline",
+        category="physical",
+        scale="50m",
+        edgecolor="None",
+        facecolor="lightgray",
+)
+ax.add_feature(continents)
+
+# Set map extent
+ax.set_extent([-180, 180, -90, 90], crs=ccrs.PlateCarree())
+
+# Create arrays with location and design of each marker
+lon = [-160, -140, -120, -100, -80, -60, -40, -20, 0, 20, 40, 60, 80, 100, 120, 140]
+lat = [-70, -60, -50, -40, -30, -20, -10, 0, 10, 20, 30, 40, 50, 60, 70, 80]
+marker = ['.', '+', '*', 'o', 'x', 's', '^', 'v', 'D', '>', '<', 'p', 'h', '8', 'X', 'd']
+
+# Draw markers on diagonal line across graph
+for x in range(len(lon)):
+    ax.scatter(lon[x], lat[x], marker=marker[x], color='blue', s=100, zorder=3)
```

Comment:
would adding fill_style='none' within this line create markers that are open like in the original? Here is a link to the resource I found this on in case you would like to add this change: https://matplotlib.org/3.1.1/gallery/lines_bars_and_markers/marker_fillstyle_reference.html

Reply:
Yes it did work! Thanks for the suggestion.